ui/apps/platform/src/Containers/NetworkGraph/types/topology.type.ts

```diff
@@ -0,0 +1,69 @@
+import { Model, NodeModel } from '@patternfly/react-topology';
+
+import { ListenPort } from 'types/networkFlow.proto';
+import { Override } from 'utils/type.utils';
+
+export type CustomModel = Override<Model, { nodes?: CustomNodeModel[] }>;
+
+export type CustomNodeModel =
+    | NamespaceNodeModel
+    | DeploymentNodeModel
+    | ExternalNodeModel
+    | ExternalEntitiesNodeModel
```

Comment:
what's the difference between ExternalNodeModel and ExternalEntitiesNodeModel here? is ExternalEntitiesNodeModel the External to Cluster node?

Reply:
<img width="283" alt="Screenshot 2022-12-05 at 4 41 25 PM" src="https://user-images.githubusercontent.com/4805485/205777770-e1c7a6ad-498a-46e6-84d3-a6a94434d606.png">

External refers to the grouping. This is based on the types defined in the `NetworkGraph` component.